pages/common/picard.md

```diff
@@ -0,0 +1,15 @@
+# picard
+
+> Next generation MusicBrainz tagging application.
+
+- Start Picard:
+
+`picard`
+
+- Open a set of files:
+
+`picard {{path/to/file1.mp3}} {{path/to/file2.mp3}} ....`
```

Comment:
I don't believe we use `...` to denote multiple files ? Mentioning 2 files is sufficient to let the user know that multiple files can be passed.

Reply:
Ah, ok